### ISSUE TITLE
Showing Unknown

### ISSUE THREAD

**Issue by ghost** [freakshock88/hass-populartimes#2]
Installed it and it shows the popular times in the "get info" dialog if you click on it with no issue, but the sensor state displays as unknown.

See here:

https://i.imgur.com/5Z5U8p5.png
https://i.imgur.com/vlt7ZWW.png

**Comment by freakshock88** [comment]
Could you post one of the place id's?

**Comment by freakshock88** [comment]
Nevermind, I found it. I tested it for 'National Mechanics'.

Somehow the populartimes library does not report current popularity for that place. 
This is the output: 
`{'id': 'ChIJOSWtVYTIxokRwl_XrRL1xz0', 'name': 'National Mechanics', 'address': '22 S 3rd St, Philadelphia, PA 19106, USA', 'types': ['bar', 'restaurant', 'food', 'point_of_interest', 'establishment'], 'coordinates': {'lat': 39.949557, 'lng': -75.146008}, 'rating': 4.4, 'rating_n': 991, 'international_phone_number': '+1 215-701-4883', 'populartimes': [{'name': 'Monday', 'data': [5, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 11, 16, 18, 17, 13, 13, 33, 50, 33, 21, 18, 13, 6]}, {'name': 'Tuesday', 'data': [2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8, 10, 10, 11, 11, 11, 15, 24, 37, 48, 53, 53, 49]}, {'name': 'Wednesday', 'data': [39, 23, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5, 9, 14, 13, 10, 15, 28, 46, 58, 59, 47, 30, 15]}, {'name': 'Thursday', 'data': [5, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8, 14, 17, 15, 15, 24, 36, 39, 34, 31, 31, 29, 21]}, {'name': 'Friday', 'data': [12, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 15, 24, 28, 24, 19, 23, 39, 56, 57, 47, 44, 57, 71]}, {'name': 'Saturday', 'data': [67, 46, 0, 0, 0, 0, 0, 0, 0, 0, 0, 14, 23, 32, 38, 39, 37, 40, 47, 55, 53, 52, 72, 100]}, {'name': 'Sunday', 'data': [92, 51, 0, 0, 0, 0, 0, 0, 0, 0, 0, 22, 36, 39, 34, 29, 26, 24, 22, 21, 20, 17, 13, 9]}], 'time_wait': [{'name': 'Monday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}, {'name': 'Tuesday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}, {'name': 'Wednesday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 15, 15, 0]}, {'name': 'Thursday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}, {'name': 'Friday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}, {'name': 'Saturday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}, {'name': 'Sunday', 'data': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}], 'time_spent': [60, 120]}`
I can't fix that unfortunately ..
